examples/acados_template/python/test/export_ode_model.py

```diff
@@ -0,0 +1,89 @@
+#
```

Comment:
Maybe we can call the function in `../pendulum_example` instead.
That would also make it clear which model is used